lib/ids_please/grabbers/base.rb

```diff
@@ -3,30 +3,38 @@
 class IdsPlease
   module Grabbers
     class Base
-
       def self.interact(links)
-        links.map { |l| self.new(l).grab_link }
+        links.map do |l|
+          new(l).grab_link
+        end
       end
 
-      attr_reader :avatar, :display_name, :username, :link, :page_source, :network_id, :data, :counts
+      attr_reader :avatar,
+                  :display_name,
+                  :username,
+                  :link,
+                  :page_source,
+                  :network_id,
+                  :data,
+                  :counts
 
       def initialize(link)
         @link = link
       end
 
-      def grab_link(link)
+      def grab_link(_link)
```

Comment:
It's an abstract class and an abstract method so you should probably have an argument in order to show what arguments the child method should have, the only thing I did was add an underscore in order to signify that it's not used for anything. Not sure why you would remove the arguments from an abstract method?